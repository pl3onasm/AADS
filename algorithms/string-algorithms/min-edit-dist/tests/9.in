retaliate
stelar
2 3 3 4 0 5
