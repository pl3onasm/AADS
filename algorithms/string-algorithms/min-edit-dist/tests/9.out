Task: source -> target
      'retaliate' -> 'stelar'

Costs: copy = 2, insert = 3, delete = 3,
       replace = 4, twiddle = 0, kill = 5

Edit sequence:
  - Source:           'retaliate'
  - Replace r -> s:   s-etaliate
  - Twiddle t and e:  ste-aliate
  - Twiddle l and a:  stela-iate
  - Insert r:         stelar-iate
  - Kill 04 chars:    stelar-
  - Result:           'stelar'

Minimum edit distance: 12
